### PR TITLE
Bug Fix for issue #26

### LINKED ISSUE #26: ng test fails.
```
$ ng test

Failed: Uncaught (in promise): Error: Template parse errors:
'md-toolbar' is not a known element:
1. If 'md-toolbar' is an Angular component, then verify that it is part of this module.
2. If 'md-toolbar' is a Web Component then add "CUSTOM_ELEMENTS_SCHEMA" to the '@NgModule.schemas' of this component to suppress this message. ("[ERROR ->]<md-toolbar color="primary">
  <span>M19G</span>
</md-toolbar>
"): AppComponent@0:0
...
```

### PR DESCRIPTION
**Pull request by suj97**
Fixed #26 . ng test, ng lint, ng e2e all working.

### CODE REVIEW

**Comment by suj97** [comment]
@kkimdev Did'nt see your commit.

**Comment by suj97** [comment]
@kkimdev Have a look. I resolved it.

File: src/index.html [hunk 1]
```diff
@@ -16,4 +16,4 @@
 <body>
   <app-root>Loading...</app-root>
 </body>
-</html>
+</html>
```
Comment (kkimdev) [inline src/index.html]: Unnecessary change, please add back the last empty line.

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -1,13 +1,12 @@
 import { TestBed, async } from '@angular/core/testing';
-
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```
Comment (kkimdev) [inline src/app/app.component.spec.ts]: nice figuring out this! :)

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
-<h1>
-  {{title}}
-</h1>
+<md-toolbar color="primary" class="mat-elevation-z4">
```
Comment (kkimdev) [inline src/app/app.component.html]: I'd add the following comment to document where mat-elevation-z4 came from.

```javascript
<!--
  4z elevation for md-toolbar per the following spec.
  https://material.io/guidelines/material-design/elevation-shadows.html
-->
```

**Comment by suj97** [comment]
@kkimdev

**Comment by suj97** [comment]
Fixed the minor changes.
